mgmt/rest/server.go

```diff
@@ -492,6 +507,41 @@ func (s *Server) addRoutes() {
 		s.r.GET("/v1/tribe/members", s.getMembers)
 		s.r.GET("/v1/tribe/member/:name", s.getMember)
 	}
+
+	// profiling tools routes
+	if s.pprof {
```

Comment:
This feature is not about profiling tools, it's about all global option we can list with `snapd -h`. Currently we can't update those options without restarting Snap.